src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -247,34 +265,34 @@ typedef struct {
     CHAR rid[MAX_STATS_STRING_LENGTH + 1]; //!< Exposes the rid encoding parameter of this RTP stream if it has been set, otherwise it is undefined
     CHAR encoderImplementation[MAX_STATS_STRING_LENGTH + 1]; //!< Identifies the encoder implementation used.
     UINT32 packetsDiscardedOnSend; //!< Total number of RTP packets for this SSRC that have been discarded due to socket errors
-    UINT32 framesSent; //!< Only valid for video. Represents the total number of frames sent on this RTP stream
-    UINT32 hugeFramesSent; //!< Only valid for video. Represents the total number of huge frames sent by this RTP stream
-                           //!< Huge frames have an encoded size at least 2.5 times the average size of the frames
-    UINT32 framesEncoded; //!< Only valid for video. It represents the total number of frames successfully encoded for this RTP media stream
-    UINT32 keyFramesEncoded; //!< Only valid for video. It represents the total number of key frames encoded successfully in the RTP Stream
+    volatile UINT64 framesSent; //!< Only valid for video. Represents the total number of frames sent on this RTP stream
```

Comment:
Seems that these are also used in atomics operations. These should also be moved up front and we should use SIZE_T as that's what the machine word will be.

If we must use UINT64 then we do need to use a LOCK. This is unfortunate but this is also the reason I had to introduce a lock in Signaling metrics to handle interlocking for the UINT64 types of variables.